packages/testing-lib/src/useTraceUpdate.ts

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-console */
+import { useEffect, useRef } from 'react';
+
+/**
+ * Used to log & debug what props caused a component to re-render
```

Comment:
Can you provide an example of how to use this?

Reply:
👍 (I'll add this in the docs as well)

```jsx
const  myComponent = ({prop1, prop2, prop3, ...rest}: SomeProps) => {

  // When debugging a rendering issue, use `useTraceUpdate` to log which prop caused a component to rerender
  useTraceUpdate({prop1, prop2, prop3, ...rest})

  return <...>
}
```